src/components/SectionTitle/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './sectionTitle.css';
+
+const SectionTitle = ({ text }) => (
```

Comment:
Since there isn't any real logic in here except rendering markup do you think we should just inline this markup wherever its being used, unless of course its being used in multiple locations? If its used in a single file we could even put this `const SectionTitle` inline, which may make it easier to grok since it will remove a level of abstraction / another file.

Reply:
Yeah I understand, however what if we want to create more sections or add more pages with titles? Isn't is a little neater to just have the section as a component that we can move around?

Reply:
Oh yeah, it definitely helps make it easier to move around and it DRYs up the code some. I think one of the hardest things in React is figuring out the balance of abstraction. I mostly mentioned this just cause it can start to get a bit difficult to keep in your mind how the DOM is created at certain points of abstraction. Anyhow, I'm cool with keeping this in 👍.